redisgears_plugin_api/src/redisgears_plugin_api/prologue.rs

```diff
@@ -0,0 +1,349 @@
+/*
+ * Copyright Redis Ltd. 2018 - present
+ * Licensed under your choice of the Redis Source Available License 2.0 (RSALv2) or
+ * the Server Side Public License v1 (SSPLv1).
+ */
+//! The validation mechanism of the user modules.
+//!
+//! Performs the basic validation and API compatibility checks.
+
+use std::collections::HashMap;
```

Comment:
Big :+1: 
My only comment here is why not to put this unit on `redisgears_core`?